README.md

```diff
@@ -149,56 +158,294 @@ This application will prompt users to fill in three or four games that they like
+
+    git clone https://github.com/aditya9729/2020-msia423-gudal-gamexplorer.git
+    
+Once cloned run to switch to the development branch:
+	
+	git checkout development
+
+### 2.Data Acquisition
+Data sources - Kaggle static files:
+   * vgsales.csv link:https://www.kaggle.com/gregorut/videogamesales
+   * steam-200k.csv link:https://www.kaggle.com/tamber/steam-video-games
+   * metacritic_games.csv link:https://www.kaggle.com/skateddu/metacritic-games-stats-20112019
+ 
+Steps to follow:
+
+   * Download these files and unzip these files till you get comma separated value files.
+   * Store these files within ./data/external/.
+   * When you run the `create_db.py` file later, it will store cleaned data in ./data/clean/
+   * Please create the clean and external directory within the data directory.
+   * It should be ./data/clean and ./data/external
+   * If you want to recreate the file storage(when there is no external and clean directory) -
+    Run the following:
+   
+    cd ./data
+    mkdir external
+    mkdir clean
+    cd external
+   * Within external directory store the files acquired and rename them.
+        * Rename 'steam-200k.csv' to 'steam.csv'.
+        * Rename 'vgsales.csv' to 'sales.csv'.
+        * Rename 'metacritic_games.csv' to 'metacritic.csv'.
+        
+### 3.Setup environment variables
+Note: You will need to be on the Northwestern VPN.
+
+Make sure you are in the root directory.
+
+You may skip this part but this may be helpful:
+
+Part1 is to create environment/ you can also run requirements.txt separately.  
+Go to part 2 to set up variables for MYSQL
+
+Part 1: Note: pip may not run within conda at times - Go to part 2 to set up variables for MYSQL
+requirements.txt may not run as pip may not be installed within conda(path not set).
+
+Create your own virtual environment.
+
+    conda create --name video_game pip
+    
+or use 
+    
+    virtualenv video_game 
+
+Activate virtual environment.
+
+    conda activate video_game
+or use:
+    
+    source video_game
+    
+Note: If you are in a virtual or base environment run the following:
+
+    pip install -r requirements.txt
+
+Part2:
+
+Edit your mysql config file accordingly
+
+    vi .mysqlconfig
+or use 
+
+    nano .mysqlconfig
+
+* Set MYSQL_USER to the "master username" that you used to create the database server.
+* Set MYSQL_PASSWORD to the "master password" that you used to create the database server.
+* Set MYSQL_HOST to be the RDS instance endpoint from the console
+* Set MYSQL_PORT to be 3306
+* SET DATABASE_NAME as the name of your rds instance if required
+
+Set up using the following within .mysqlconfig(please remove the caret '<' '>' signs:
+
+    export MYSQL_USER=<YOUR_USERNAME>
+    export MYSQL_PASSWORD=<YOUR_PASSWORD>
+    export MYSQL_HOST=<PASTE_YOUR_HOST_HERE>
+    export MYSQL_PORT=<3306>
+    export DATABASE_NAME=msia423asg5718
+
+
+Set the environment variables in your ~/.bashrc
+
+    echo 'source .mysqlconfig' >> ~/.bashrc
```

Comment:
when you echo your username and password into `~/.bashrc`, the env variables become persistent (i.e. accessible in every bash session). This may not be what the user wants. it is probably better to just set them temporarily using `source .mysqlconfig`.

Reply:
These are temporary within .mysqlconfig. If I'm not wrong, one won't be able to access environment variables if it is not included within bashrc or a environment file. Do let me know what you think.